README.md

```diff
@@ -93,26 +130,35 @@ source .venv/bin/activate
 pip install -r requirements.txt
 ```
 
-In CML or CDSW, no virtual env is necessary. Instead, inside a JupyterLab Session (with at least 2 vCPU / 16 GiB Memory), simply run through the Setup cells in the Jupyter Notebook. 
+-----------
 
-### Running the evaluation script
 
-The [scripts/evaluate.py](scripts/evaluate.py) script requires the [scripts/config.txt](scripts/config.txt) file which handles all the input arguments. First specify your choices in the config file, and then run the following in an open Session: `!python3 scripts/evaluate.py @scripts/config.txt`
+## Execution 
 
-Alternatively, this script can also be run automatically with the **Jobs** abstraction by first clicking on **Jobs**, then **New Job**, and then selecting `scripts/evaluate.py` under **Script**. Enter `@scripts/config.txt` under **Arguments** and ensure that the job is given at least 2vCPU/16GiB of resources. Then click **Create Job**. You can now run the job as often as you like or schedule the job at your convenience. 
+Launch a JupyterLab Session with at least 16 GiB of memory and 2 vCPUs, on CML, or with similar resources on your own system.
```

Comment:
A JupyterLab Session is for running the Jupyter Notebook, so this line should probably be under the ##Jupyter notebook section below. 

The evaluation script would be run on a standard Session (or via the Jobs abstraction)

Reply:
Updated.